core_lib/src/structure/camera.h

```diff
@@ -51,11 +52,16 @@ class Camera : public KeyFrame
 
     bool operator==(const Camera& rhs) const;
 
+    void setEasingType(CameraEasingType type) { mEasingType = type; }
+    CameraEasingType getEasingType() { return mEasingType; }
```

Comment:
CameraEasingType getEasingType() { return mEasingType; }
```suggestion
    CameraEasingType getEasingType() const { return mEasingType; }
```